pkg/controller/v1beta1/inferenceservice/controller.go

```diff
@@ -105,6 +107,12 @@ func (r *InferenceServiceReconciler) Reconcile(req ctrl.Request) (ctrl.Result, e
 		return reconcile.Result{}, errors.Wrapf(err, "fails to reconcile ingress")
 	}
 
+	//Reconcile modelConfig
+	configMapReconciler := modelconfig.NewModelConfigReconciler(r.Client, r.Scheme)
```

Comment:
Can we move this to predictor as it is only relevant for model server

Reply:
There won't be any InferenceService without a predictor right?

Reply:
let's add a check only to create the config if storageUri is nil, so user are not confused creating an empty config

Reply:
Done